app/views/flood_risk_engine/company_address_manual_forms/new.html.erb

```diff
@@ -20,15 +21,17 @@
 
       <p class="govuk-body">
         <%= t(".preset_postcode_label") %>
-      <br/>
-      <span class="postcode govuk-!-font-weight-bold">
-        <%= @company_address_manual_form.postcode %>
-      </span>
-      <%= link_to(t(".postcode_change_link"), back_company_address_manual_forms_path(@company_address_manual_form.token)) %>
+        <br/>
+        <span class="postcode govuk-!-font-weight-bold">
```

Comment:
I don't think we have a css class for '.postcode' anymore, so that can probably go?

Reply:
We do in the host apps – it adds a bit of extra padding between the postcode and the change link